src/main.ts

```diff
@@ -712,7 +724,7 @@ export default class WhatsApp {
 
             setTimeout(this.keepAlive.bind(this), 20 * 1000);
 
-            if (this.keysPath) {
+            if (this.parameters.keysPath) {
```

Comment:
```suggestion
            if (this.opts.keysPath) {
```
Update property again